### PR TITLE
removed validations from name field

### LINKED ISSUE #947: Cannot register with apostrophies in name
# Bug Report
## What is the current behavior?
Doing names like: N'golo or O'Hara are impossible due to incorrect validation of the name inputs.

## What is the expected behavior?
The name fields should have no validation.

## What steps did you take to get this behavior?
Noticed a Cypress test failing with randomly generated data

### Screenshots
Screen Shot 2019-11-18 at 2 13 01 PM

### PR DESCRIPTION
**Pull request by aasifkhan7**
# Description of changes

# Issue Resolved

Fixes #947

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit 47889cd6b57dae789e84cd867c8e25789a853c9c

https://deploy-preview-957--operation-code-storybook.netlify.com

File: components/RegistrationForm/RegistrationForm.js [hunk 1]
```diff
@@ -28,14 +28,8 @@ const registrationSchema = Yup.object().shape({
...
-  firstName: Yup.string()
-    .required(validationErrorMessages.required)
-    // eslint-disable-next-line no-useless-escape
-    .matches(/^[a-zA-Zà-žÀ-Ž \-]+$/, validationErrorMessages.name),
-  lastName: Yup.string()
-    .required(validationErrorMessages.required)
-    // eslint-disable-next-line no-useless-escape
-    .matches(/^[a-zA-Zà-žÀ-Ž \-]+$/, validationErrorMessages.name),
+  firstName: Yup.string(),
+  lastName: Yup.string(),
```
Comment (kylemh) [inline components/RegistrationForm/RegistrationForm.js]: We still want these listed as required! Also, you'll want to remove some code in test files that assert `validationErrorMessages.name` is rendered.

Comment (aasifkhan7) [reply components/RegistrationForm/RegistrationForm.js]: I marked these fields as required. Also, I was not able to find any tests that assert that `validationErrorMessages.name` is rendered when the input fields do not match the regex given

**Comment by kylemh** [comment]
Did you skip the precommit hooks? Looks like the linter is yelling at you!

**Comment by aasifkhan7** [comment]
I must have missed those. My bad. Fixed the linting errors.

**Pull request merged**